INSTALL.md

```diff
@@ -519,13 +503,6 @@ which can reduce this limit even further.
 
 We therefore recommend to always build and use GAP in 64-bit mode.
 
-### Recompilation fails or the new binary crashes.
```

Comment:
While I won't claim this never happens, I think overall it is muuuuch rarer than it was with the build system used in GAP 4.8 and before, and as such it seems highly questionable to cover it here.

Reply:
I agree